src/components/Headline/HeadlineTextStyles.ts

```diff
@@ -0,0 +1,113 @@
+import { remCalc } from '../../utils/remCalc';
+
+export enum HeadlineTextTransform {
```

Comment:
Sure you want to export it as enum - and not const enum? Difference is that const enum will result in a replacement at compile-time, while enum will have a runtime cost.